File: PythonActividad1Poo/Domain/Gimnasio.py
class Gimnasio:
    def __init__(self, nombre, ubicacion, numero_maquinas):
        self.nombre = nombre
        self.ubicacion = ubicacion
        self.numero_maquinas = numero_maquinas
        self.usuarios = []

    def agregar_usuario(self, usuario):
        self.usuarios.append(usuario)

    def set_entrenador(self, entrenador):
        self.entrenador = entrenador

    def get_usuarios(self):
        return self.usuarios

    def __str__(self):
        return f"Gimnasio: {self.nombre}, Ubicación: {self.ubicacion}, Máquinas: {self.numero_maquinas}, Entrenador: {getattr(self, 'entrenador', 'N/A')}"

File: JavaScriptActividad1Poo/public_html/co.edu.udec.s2.Ejercicio29/Gimnasio.js
class Gimnasio {
    constructor(nombre, ubicacion, numeroMaquinas) {
        this.nombre = nombre;
        this.ubicacion = ubicacion;
        this.numeroMaquinas = numeroMaquinas;
        this.usuarios = [];
    }

    agregarUsuario(usuario) {
        this.usuarios.push(usuario);
    }

    toString() {
        let result = `Gimnasio: ${this.nombre}, Ubicación: ${this.ubicacion}, Máquinas: ${this.numeroMaquinas}`;
        return result;
    }
}